### ISSUE TITLE
[DOCS] Add Docs Recipe on “ Adding custom connectors”

### ISSUE THREAD

**Issue by metalboyrick** [Scaffold-Stark/ss2-docs#59]
## Issue Overview

Add a recipe entry on how to create custom wallet connectors. You can refer to the `BurnerWalletConnector` as an example.

## Proposed Solutions or Ideas

- Please make sure that you clone the https://github.com/Scaffold-Stark/scaffold-stark-2 repo and understand how the project works
- Refer to `BurnerWalletConnector` code.
- Try creating your own "wallet connector".

**Comment by vestor-dev** [comment]
i'm a frontend dev and blockchain dev 
May I try my hand at this?
i would love to work on this and get it done

**Comment by CollinsC1O** [comment]
May I take this issue on?

**Comment by Supa-mega** [comment]
Is this issue still available?

**Comment by jedstroke** [comment]
Can I try solving this issue?

**Comment by abdegenius** [comment]
May I try my hand at this?

**Comment by 1nonlypiece** [comment]
Hi ,
I’d like to work on this issue as I have experience in creating and maintaining clear and user-friendly documentation..

ETA:
 I estimate completing this task in about 12-24 hours. Let me know if I can get started!

**Comment by Jagadeeshftw** [comment]
Hi,
I’d love to work on this issue as I have strong experience in developing comprehensive and user-focused documentation. Please assign this to me, and I’ll deliver an efficient and secure PR within 12-24 hours.

**Comment by Jagadeeshftw** [comment]
Hi,
I’d love to work on this issue as I have strong experience in developing comprehensive and user-focused documentation. Please assign this to me, and I’ll deliver an efficient and secure PR within 12-24 hours.

**Comment by greatest0falltime** [comment]
Hi,

I’d be thrilled to work on this issue as I have a strong background in crafting well-structured and accessible documentation.

ETA:  
I expect to have a working solution ready within the next 24 hours. Kindly let me know if I can proceed!  

Looking forward to contributing!

**Comment by emmz3230** [comment]
Could I take on this issue?

I am a front-end developer specializing in responsive web design,
device-compatible email templates, 3D web integration,
and technical writing on programming topics. I also contribute
to open-source documentation and help build documentation sites.

**Comment by mimiprosper** [comment]
I would like to take this task. Am a web 2 frontend developer, a technical writer, a solidity smart contract developer/auditor and a cairo smart contract developer. Am a graduate of Starknet Cairo Bootcamp. I have successfully done tasks assigned to me in OnlyDust. 
If assigned this i would complete it within 2 days. 

Here are some articles i have written: https://medium.com/@emma.onyedika.okeke

**Comment by pheobeayo** [comment]
Could I take over this issue?

I am a Web3 front-end developer skilled in technical documentation and software development. I greatly understand technologies like JavaScript, Typescript, Cairo, Rust and Solidity. I have contributed successfully to open-source projects like Stark Quest and Speedrun scaffold-stark
My Approach includes:
- Reviewing of existing Documentation:
by thoroughly reading the current documentation, including README files, code comments, and any other relevant documentation provided in the repository.
Identify areas that are unclear, outdated, or missing crucial information that would be of benefit to users and developers
- Make note of changes and define all code snippets and technical terms in simple terms.
- Create a Pull Request (PR).
- Request for a Review.
- Make changes based on the review of the PR.

**Comment by ShantelPeters** [comment]
Mind if I try this one?

**Comment by Amarjeet325** [comment]
Can I take care of this issue?

**Comment by mimisavage** [comment]
Could I try solving this?

**Comment by TheSQLguy** [comment]
To address this issue, I would:

Clone the Scaffold-Stark repository and review its structure to understand how wallet connectors are implemented.
Study the BurnerWalletConnector code as a reference to learn the implementation details and patterns.
Create a custom wallet connector and document the process in a recipe entry, including step-by-step instructions and code snippets to help explain.
Ensure that the recipe follows the standards of the project and is readable to another developer.

**Comment by BrunoAmbricca** [comment]
I would like to work on this issue

**Comment by od-hunter** [comment]
Hi, can I be assigned this please? I am a blockchain developer and I'm also good at documentations. Below are some of the documentations I've done that were merged:

https://github.com/dojoengine/book/pull/308,
https://github.com/keep-starknet-strange/joyboy/pull/143,
https://github.com/Scaffold-Stark/ss2-docs/pull/31,
https://github.com/horuslabsio/Starknet-Scaffold/pull/94,
https://github.com/Hyle-org/examples/pull/15,

Please I’m ready to work, assign me please.

**Comment by tasneemtoolba** [comment]
can I work on this task? I'm very interested to be a contributor after finishing starknet speedrun challenges 1-2 months ago, after starknet cohort with H.E.R DAO)

**Comment by emarc99** [comment]
Could I take on this issue? I have research and writing experience like https://emarc.hashnode.dev/starknet-development-with-docker-a-streamlined-setup-for-cairo-builders.
ETA 2 days

**Comment by olisaagbafor** [comment]
I’m interested in this one.

**Comment by michojekunle** [comment]
I'd like to handle this task.

**Comment by metalboyrick** [comment]
all yours @mimiprosper !

**Comment by metalboyrick** [comment]
hey @mimiprosper , any updates here?

**Comment by metalboyrick** [comment]
gm @mimiprosper , any updates here?

**Comment by Jagadeeshftw** [comment]
Can this be assigned to me @metalboyrick? I'm already familiar with wallet connectors.

**Comment by mimiprosper** [comment]
@metalboyrick 
There is an update. Am working on it I would make a pull request now

**Comment by mimiprosper** [comment]
Hi 
@metalboyrick 
Am aware of that. It would be ready before deadline, Sunday.

**Comment by Nadai2010** [comment]
Perfect, especially in case you PR need to make changes that may be within the margin, thank you.

**Comment by metalboyrick** [comment]
Yes, please do consider the time window for reviews too, thanks!

**Comment by mimiprosper** [comment]
@metalboyrick 
I am aware of the review process.
We conclude everything today

**Comment by mimiprosper** [comment]
@metalboyrick 
Please check PR. 
I wait for your response
Thanks.

**Comment by mimiprosper** [comment]
Am working on it right

**Comment by mimiprosper** [comment]
GM @metalboyrick

Please do you have materials that can help? There are limited materials/no material about custom wallet connector on starknet. If you have any material that can helpful please send. BurnerWalletConnector not enough resource. Thanks.

**Comment by mimiprosper** [comment]
@metalboyrick 
Pls check update.
Thanks.

**Comment by od-hunter** [comment]
Hi @Nadai2010 , can I help with this please?
